src/components/questionnaire/CodeInput.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { StyleSheet, TextInput, TouchableOpacity, View } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import COLORS from '../../assets/colors';
+import FONTS from '../../assets/fonts';
+
+const CodeInput = () => {
+  const [code, setCode] = useState<String>('');
+
+  return (
+    <View style={styles.codeForm}>
+      <TextInput
+        style={styles.codeInput}
+        placeholder="code invoeren..."
```

Comment:
Weet niet of we hier evt een constant van willen maken zodat t makkelijk op 1 locatie aan te passen is?

Reply:
Ja maybe, maar aangezien we dit waarschijnlijk alleen op deze plek gebruiken is dat dan niet overbodig?

Reply:
Ja dan maakt het opzich niet heel veel uit, meer voor de fancy :D